disk/datadog_checks/disk/disk.py

```diff
@@ -61,6 +62,10 @@ def check(self, instance):
         """Get disk space/inode stats"""
         if self._tag_by_label and Platform.is_linux():
             self.devices_label = self._get_devices_label()
+        if not isinstance(self._min_disk_size, int):
```

Comment:
No need, at this point it would have already failed. Also floats should be accepted